hibernate-core/src/main/java/org/hibernate/usertype/DynamicParameterizedType.java

```diff
@@ -33,6 +35,10 @@ interface ParameterType {
 
 		Class<?> getReturnedClass();
 
+		default Type getReturnedJavaType() {
```

Comment:
Last change and then this can be merged :)

```suggestion
		@Incubating
		default Type getReturnedJavaType() {
```

Reply:
Done.